charts/generic-service/README.md

```diff
@@ -184,3 +184,65 @@ Job progress can then be seen by running `kubectl logs -f` on the newly created
 
 The last successful restore information is stored in a `restore_status` table in pre-production.
 To find out when the last restore ran connect to the pre-production database and view the contents of the table.
+
+### Scheduled downtime
+
+For cost saving purposes, MOJ Cloud Platform provides an option to shut down RDS databases overnight in non-production
+environment.
+[Check the user guide for more information](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/deploying-an-app/relational-databases/create.html#non-production).
+
+In addition to shutting down the database, this chart also provides an option to schedule shutdown and startup of pods.
+
+#### Service Account
+
+To enable this feature, you first need to add a `scheduled-downtime-serviceaccount` Service Account to your namespace,
+with permissions to scale your deployment.
+
+Example: [scheduled-downtime.tf](https://github.com/ministryofjustice/cloud-platform-environments/blob/23fdd1a1b13e1ede43f778b7be6c8765939f27a0/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-probation-integration-services-dev/resources/scheduled-downtime.tf)
+
+```terraform
```

Comment:
Just wondering why you chose to use terraform to create the service account - instead of putting some manifests in the with the other configuration [here](https://github.com/ministryofjustice/cloud-platform-environments/blob/23fdd1a1b13e1ede43f778b7be6c8765939f27a0/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-probation-integration-services-dev/) ?

Reply:
I think your solution is good. I guess we should align with whatever the CP team recommend going forward.